bedrock/mozorg/views.py

```diff
@@ -234,7 +234,40 @@ class ContributeIndex(l10n_utils.LangFilesMixin, TemplateView):
     template_name = 'mozorg/contribute/index.html'
 
 
+class ContributeSignupNew(TemplateView):
+
+    def get_template_names(self):
+        variant = self.request.GET.get('v', '')
+        template = 'mozorg/contribute/signup-new.html'
+
+        if variant == '2':
+            template = 'mozorg/contribute/signup-alternate.html'
+
+        return [template]
+
+
+class ContributeTaskView(TemplateView):
+
+    def get_template_names(self):
+        variant = self.request.GET.get('v', '')
+        task = filter(None, self.request.path.split('/'))[-1]
+
+        # if the variant was not set, this is a v1 variant
+        if not variant:
+            variant = '1'
+
+        template = 'mozorg/contribute/tasks/v{0}/{1}.html'.format(variant, task)
+        return [template]
+
+
 def contribute_signup(request):
+    # for the new signup landing pages, a variant will be specified
+    # using the URL parameter v
+    variant = request.GET.get('v', '')
+
+    if variant:
```

Comment:
For safety's sake, suggest changing this to:

```
if variant in ['1', '2']:
```

Also suggest wrapping the rest in an `else` to clarify logic flow.